meta-mel/recipes-support/consolekit/consolekit_%.bbappend

```diff
@@ -1,5 +1,10 @@
 FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
 
 SRC_URI_append = " \
-    file://0001-Ensure-filesystems-are-still-mounted-when-consolekit.patch \
+    file://add-dependency-on-var-volatile.mount-for-shutdown.patch \
 "
+
+do_install_append() {
+    install -d ${D}${sbindir}
+    install -m 755 ${S}/data/console-kit-shutdown-handler ${D}${sbindir}
```

Comment:
Not intended for upstream, since this issue is specific to having /var/log disappearing before being able to write to it on shutdown.
So, you mean it would be better to just let the makefile do what it was doing, then removing the files it creates through the recipe instead of patching the makefile ?

Reply:
@bboccoqq, yup I believe that would be a more feasible approach and would allow anyone looking at the changes easily understand as the changes will be happening in a single place i.e. the recipe.

Reply:
OK, re-implemented this way. You're right it's cleaner.

I decided to patch the package/Makefile the first time around since it was taking care of consuming the *.service.in files.